Add Promscale with Timescale cloud docs (#1658)
timescale/docs: _partials/_promscale-send-data.md

> After you have installed Promscale, you can ingest data using these options:

Reviewer: ```suggestion
When you have installed Promscale, you can ingest data in a few different ways:
```

> *   To migrate existing Prometheus data use [Prom-migrator][promscale-migrator].
> *   To ingest the current incoming data, see the [send data][send-data] section.

Reviewer: ```suggestion
*   Migrate existing Prometheus data with [Prom-migrator][promscale-migrator].
*   Ingest incoming data. For more information, see the [send data][send-data] section.
```
